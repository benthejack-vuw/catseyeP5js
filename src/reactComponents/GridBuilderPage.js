import React from 'react'

const GridBuilderPage = () => (
    <div>
        GRID BUILDER    
    </div>
)

export default GridBuilderPage;