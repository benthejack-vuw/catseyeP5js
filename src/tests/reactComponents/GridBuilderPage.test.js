import React from 'react';
import { shallow } from 'enzyme';
import GridBuilderPage from '../../reactComponents/GridBuilderPage';

test('should render GridBuilderPage correctly', ()=>{
    const wrapper = shallow(<GridBuilderPage startLogout={()=>{}}/>);
    expect(wrapper).toMatchSnapshot();
})