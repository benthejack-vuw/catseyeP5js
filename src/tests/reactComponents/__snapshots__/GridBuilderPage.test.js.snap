// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render GridBuilderPage correctly 1`] = `<div />`;
